### PR TITLE
APPS-1028 Fix cleanup function to support full id when building from

### PR DESCRIPTION
**Pull request by YuxinShi0423**

### CODE REVIEW

**Comment by r-i-v-a** [comment]
Testing it here https://github.com/dnanexus/dxCompiler/pull/254 @YuxinShi0423

File: src/python/dxpy/workflow_builder.py [hunk 1]
```diff
@@ -71,7 +71,7 @@ def _fetch_spec_from_dxworkflowjson(src_dir, json_file_name, parser):
 
 def _cleanup_empty_keys(json_spec):
     import re
-    clean_json = re.sub('\"\w*\": (\{\}|\"\"|\[\])(\,|)\s*','',json.dumps(json_spec)).replace(", }","}")
+    clean_json = re.sub('\"\w*\": (null|\{\}|\"\"|\[\])(\,|)\s*','',json.dumps(json_spec)).replace(", }","}")
```
Comment (r-i-v-a) [inline src/python/dxpy/workflow_builder.py:74]: Can this be somehow covered by a unit test? @YuxinShi0423

**Review by r-i-v-a — CHANGES_REQUESTED** [review]
Can this be somehow covered by a unit test?

**Comment by YuxinShi0423** [comment]
https://jenkins-vstg.internal.dnanexus.com/job/dx-toolkit/job/dxpy-branch-integration-tests-on-staging-20.04-python3/115/console
Unit test passed,

**Review by kpjensen — APPROVED** [review]

**Pull request merged**